processors/numberlines.go

```diff
@@ -0,0 +1,50 @@
+package processors
+
+import (
+	"fmt"
+	"strings"
+)
+
+// / Prepends consecutive number to each line of input
+type LineNumberer struct{}
+
+func (p LineNumberer) Name() string {
+	return "number-lines"
+}
+
+func (p LineNumberer) Alias() []string {
+	return []string{"nl", "number-lines"}
+}
+
+func (p LineNumberer) Transform(data []byte, _ ...Flag) (string, error) {
+	var s = string(data[:])
+	var counter = 1
+	var result = ""
+	var err error
+	for line := range strings.Lines(s) {
+		if line == "\n" {
+			result += line
+		} else {
+			result += fmt.Sprintf("%d. %s", counter, line)
+			counter += 1
+		}
+
+	}
+	return result, err
```

Comment:
Don't add an err variable, simply do this

```suggestion
	return result, nil
```

Reply:
Good catch. Fix in 652c4c6